src/main/java/nl/novi/techiteasy/exceptions/RecordNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package nl.novi.techiteasy.exceptions;
+
+public class RecordNotFoundException extends RuntimeException{
+    private static final long serialVersionUID = 1L;
```

Comment:
Weet je wat een serialVersionUID is en waarom je het zou moeten gebruiken of niet? https://stackoverflow.com/questions/285793/what-is-a-serialversionuid-and-why-should-i-use-it